tests/manage/z_cluster/nodes/test_disk_failures.py

```diff
@@ -66,6 +69,34 @@ def detach_volume_and_wait_for_attach(self, nodes, data_volume, worker_node):
                 f"node {worker_node}"
             )
 
+    def clear_warnings_and_leftovers(self):
```

Comment:
please move this function outside the test file

Reply:
Okay, will do

Reply:
it can be a good function to be added to helpers.py and it can be renamed so what like `clear_crash_warning_and_osd_removal_leftovers` please rephrase if needed

Reply:
Yes, sounds good @prsurve